ChangeLog

```diff
@@ -9,6 +9,7 @@ Bugfix
    * Fix memory leak in mbedtls_ssl_set_hostname() when called multiple times.
      Found by projectgus and jethrogb, #836.
    * Fix usage help in ssl_server2 example. Found and fixed by Bei Lin.
+   * Parse signature algorithm extension when renegotiating.
```

Comment:
Could you expand a bit on the consequences so that users can recognize that this relates to a failure that they have, or could potentially, experience? Something like "Parse signature algorithm extension when renegotiating. Previously, renegotiating handshakes would only accept signatures using SHA-1, or fail if the latter was disabled."

Reply:
@hanno-arm I updated the ChangeLog, but actually, even if SHA1 is not disabled and handshake succeeds, the behavior is wrong